app/views/projects/_approve_form.html.erb

```diff
@@ -4,18 +4,18 @@
 <div class="container">
   <div class="field">
     <label for="project_directory" class="form-label">Confirm Project Directory</label>
-    <input class="form-directory" type="text" name="project_directory-prefix" disabled readonly value="<%= @project.project_directory_parent_path %>"></input> /  
-    <input class="form-directory-confirm" type="text" name="project_directory" required placeholder=<%= "#{@project.project_directory_short}"%>></input>  
+    <input class="form-directory" type="text" name="project_directory-prefix" required value="<%= @project.project_directory_parent_path %>"></input> /  
```

Comment:
I noticed this small bug where after the project gets approved, the projects `directory_parent_path` gets set to `.`
![Screenshot 2024-05-09 at 10 11 03 AM](https://github.com/pulibrary/tiger-data-app/assets/54775395/403bcfc0-97de-4b26-8228-e88bc615a80e)
 you can see if you navigate back to the approve page after approving it.